packages/shared/styles/components/organisms/SfBottomNavigation.scss

```diff
@@ -1,59 +1,80 @@
 @import "../../helpers";
 
+
+.sf-bottom-navigation {
+  box-sizing: border-box;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: var(--bottom-navigation-z-index, 1);
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-end;
+  width: 100%;
+  height: var(--bottom-navigation-height, 3.75rem);
+  padding: var(--bottom-navigation-padding, 0 var(--spacer-sm));
+  background: var(--bottom-navigation-background, var(--c-white));
+  box-shadow: var(
+    --bottom-navigation-box-shadow,
+    0px -2px 10px rgba(154, 154, 154, 0.15)
+  );
+}
+
 .sf-bottom-navigation-item {
   --icon-color: var(--c-gray-variant);
-  position: relative;
   top: var(--bottom-navigation-item-top);
-  margin: var(--bottom-navigation-item-margin);
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  font: var(
+  align-self: var(--bottom-navigation-item-align-self);
+  margin: var(--bottom-navigation-item-margin);
+  padding: var(--bottom-navigation-item-padding, var(--spacer-xs) 0 var(--spacer-xs) 0);
+  color: var(--bottom-navigation-item-color, var(--c-gray-variant));
+  
+  @include font(
     --bottom-navigation-item-font,
-    font(
-      --bottom-navigation-item,
-      var(--body-font-weight-primary),
-      var(--font-size-small),
-      1.6,
-      var(--font-family-primary)
-    )
+    var(--font-light),
+    var(--font-xs),
+    1,
+    var(--font-family-primary)
   );
-  color: var(--bottom-navigation-item-color, var(--c-gray-variant));
+
+
+  &  .sf-circle-icon {
+    --button-size: 4.125rem;
+  }
+  
   &__label {
-    line-height: 1.6;
+    margin: var(--bottom-navigation-item-label-margin);
   }
+
   &--active {
+    --icon-color: var(--c-black);
     --bottom-navigation-item-color: var(--c-black);
-    --bottom-navigation-item-font-weight: var(--body-font-weight-secondary);
-    --icon-color: var(--bottom-navigation-item-color--active);
+    --bottom-navigation-item-font-weight: var(--font-normal);
   }
+
   &--has-margin {
-    --bottom-navigation-item-margin: 6px 0 0 0;
+    --bottom-navigation-item-label-margin: var(--spacer-xs) 0 0 0;
   }
+
   &--floating {
-    --bottom-navigation-item-top: -1.5rem;
     --icon-color: var(--c-white);
     &:not(:hover) {
```

Comment:
can we use it without `&:not(:hover) {}`? maybe we should move it to SfCircleButton?